vars/Debian.yml

```diff
@@ -40,6 +40,7 @@ system_packages:
   - python3-psycopg2
   - python3-setuptools
   - python3-pip
+  - python3-urllib3
```

Comment:
we don't need to install this package by default using the default package manager if the pip installation method is selected urllib3 will be installed along with all other [requirements](https://github.com/vitabaks/postgresql_cluster/blob/master/files/requirements.txt)

I think this package does not have to be explicitly specified even if the rpm/deb installation method is selected since the package manager itself must identify and install the dependent packages.

Or, move this package to the `patroni_packages` section.